packages/ui/src/components/va-progress-bar/VaProgressBar.vue

```diff
@@ -47,12 +53,12 @@ export default defineComponent({
     buffer: { type: Number, default: 100 },
     rounded: { type: Boolean, default: true },
     reverse: { type: Boolean, default: false },
+    slotInside: { type: Boolean, default: false },
```

Comment:
Which slot?

How about to rename it in `contentInside`, `percentInside` or simply `inside`?

Reply:
I choose `contentInside ` ))